tracker/templates/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /all-incidents/?*
+Allow: /all-incidents/?page=*
+
+Sitemap: https://pressfreedomtracker.us/sitemap.xml
```

Comment:
Could I make a recommendation to add a [`Crawl-Delay`](https://en.wikipedia.org/wiki/Robots.txt#Crawl-delay_directive) directive, just to help aid the performance matter?